simpleflow/canvas.py

```diff
@@ -72,11 +109,20 @@ def sync_state(self):
 
     def sync_result(self):
         self._result = []
+        exceptions = []
+        have_exceptions = False
```

Comment:
Necessary? Clearer to have just a `if any(exc for exc in exceptions):` at the end?